client/js/screens/About/styles.js

```diff
@@ -0,0 +1,31 @@
+import { StyleSheet } from "react-native";
+import theme from "../../config/globalStyles";
+
+const styles = StyleSheet.create({
+  container: {
+    top: 40,
+    marginLeft: 20,
+    marginRight: 20
+  },
+  h2: {
+    fontSize: theme.titeFontSize,
+    color: theme.black,
+    fontFamily: theme.titleFont
+  },
+  p: {
+    // fontFamily: theme.bodyFont,
+    fontSize: theme.commonFontSize,
+    paddingTop: 10,
+    lineHeight: 25
+  },
+  image: {
+    marginTop: 60,
```

Comment:
We should not put this much margin. Use flexbox properties. resizeMode: "contain".